spec/lib/krikri/aggregation_entity_behavior_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Krikri::AggregationEntityBehavior do
+
+  before do
+    # The harvest activity is the activity that generated the entities upon
+    # which the mapping activity below has acted.  I suppose we could mock the
+    # call that will happen to EntityConsumer#assign_generator_activity! when
+    # Krikri::Mapper::Agent is instantiated, but that seems like jumping
+    # through more hoops and making this less comprehensible than it is now.
+    DatabaseCleaner.clean_with(:truncation)
+    create(:krikri_harvest_activity)
+  end
+
+  describe '#entities' do
+
+    let(:mapping_activity_uri) do
+      (RDF::URI(Krikri::Settings['marmotta']['ldp']) /
```

Comment:
Regarding the dummy URI, `#rdf_subject` responds with a URI that includes the hostname of whatever host is running the tests, so I was trying to make the tests as portable as possible, and I'm sure I didn't realize about the use of `double` at the time.